src/style.css

```
@@ -0,0 +1,66 @@
+body {
+  margin: 0;
+  
+}
+
+.stars {
+  display: flex;
+}
+
+/* .container {
+  margin-top: 2px;
+  margin-left: 2px;
+} */
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  
+
+}
```

The reason of failed test is that background-image wasn't centered. Also you can set background-image right here, you don't need additional classes below. Also see below how you can set margin-right for all elements except of the last child.
```suggestion
.stars__star {
  width: 16px;
  height: 16px;
  background-repeat: no-repeat;
  background-position: center;
  background-image: url('./images/star.svg');
}
```